src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="form-box">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Name:
+            <input 
+              type="text"
+              name="username"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              min="1"
+              max="100"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Full date of birth:
+            <input 
+              type="date"
+              name="data"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="accept"
+            >
+          </label>
+        </div>
+  
+      </fieldset>
+
+
+      <fieldset class="form-box">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+              multiple
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Password:
+            <input 
+              type="password"
+              name="password"
+              required
+              minlength="8"
+              maxlength="20"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
```

Comment:
Please add separte labels for each radio.
label
  radio yes
label

label
  radio no
label

Reply:
I don't know how to correct it in the comments, so I'll try this way.

<div class="form-field">
          Do you love cats?
          <label>
            <input 
              type="radio"
              name="cats"
              value="yes"
              checked
            >
            Yes
          </label>
          <label>
            <input 
              type="radio"
              name="cats"
              value="no"
            >
            No
          </label>
        </div>

Reply:
I don't know how to correct it in the comments, so I'll try this way.

<div class="form-field">
          Do you love cats?
          <label>
            <input 
              type="radio"
              name="cats"
              value="yes"
              checked
            >
            Yes
          </label>
          <label>
            <input 
              type="radio"
              name="cats"
              value="no"
            >
            No
          </label>
        </div>

Reply:
`I don't know how to correct it in the comments, so I'll try this way.

<div class="form-field">
          Do you love cats?
          <label>
            <input 
              type="radio"
              name="cats"
              value="yes"
              checked
            >
            Yes
          </label>
          <label>
            <input 
              type="radio"
              name="cats"
              value="no"
            >
            No
          </label>
        </div>
`